### DISCUSSION TITLE
Update of fmriprep to version 23.2.1

### DISCUSSION THREAD

**Discussion by Beatrizfg20** [neurodesk/neurodesk.github.io-hugo#418] [Q&A] [answered]
Hello, Could anyone update fmriprep to version 23.2.1? It would be really useful for us to process my task fmri images including another freesurfers outputs (--derivatives command)
Thank you in advance!!

**Comment by stebo85** [comment]
Dear @Beatrizfg20,

You can propose an update through a pull request here next time: https://github.com/NeuroDesk/neurocontainers/blob/master/recipes/fmriprep/build.sh

I now did that and it will be online in a few hours :) I let you know once it's available.

Thank you
Steffen

**Comment by stebo85** [comment, marked as answer]
Dear @Beatrizfg20 

fmriprep is now updated. You can either use the new version through the module system:
<img width="678" alt="image" src="https://github.com/NeuroDesk/neurodesk.github.io/assets/4021595/d4be5ec6-5510-4554-b387-6e68666c0d95">

or by clicking update in the Application menu:
<img width="667" alt="image" src="https://github.com/NeuroDesk/neurodesk.github.io/assets/4021595/a9ec6829-c689-4298-86bd-da926f71c983">
